### DISCUSSION TITLE
Linker Error LNK1169 Multiply defined symbols with two different versions

### DISCUSSION THREAD

**Discussion by Tastaturtaste** [gfx-rs/wgpu#3211] [Q&A] [answered]
I am currently working on a program using wgpu 0.14.0 and pixels 0.10.0 directly. This version of pixels in turn uses wgpu 0.13.1 internally. The compilation works fine, but linking leads to the error in the title of my question. 

I know one possibility to work around this issue is to reduce my version of wgpu to the same version as pixels, such that only one version is present in the program. But normally it should be possible to use multiple different versions of the same crate in one program without issue, as long as the types are not mixed. 

Is it possible to resolve this linker error without changing my version of wgpu?

**Comment by cwfitzgerald** [comment, marked as answer]
Not currently, though definitely something we want to see fixed https://github.com/gfx-rs/wgpu/issues/3105
